Project/Source/Components/ComponentParticleSystem.cpp

```diff
@@ -2159,6 +2159,15 @@ void ComponentParticleSystem::StopChildParticles() {
 	}
 }
 
+void ComponentParticleSystem::SetParticlesPerSecondChild(float2 particlesPerSecond) {
+	SetParticlesPerSecond(particlesPerSecond);
+	for (GameObject* currentChild : GetOwner().GetChildren()) {
+		if (currentChild->GetComponent<ComponentParticleSystem>()) {
+			currentChild->GetComponent<ComponentParticleSystem>()->SetParticlesPerSecond(particlesPerSecond);
```

Comment:
Is this supposed to be recursive or just for the immediate childs of the object?